client-side/src/Components/TableComponent.scss

```diff
@@ -0,0 +1,77 @@
+@import url('https://fonts.googleapis.com/css2?family=Nanum+Gothic&display=swap');
+
+.table {
+    background-color: rgb(255, 247, 247);
+    margin-top: 10%;
+    margin-left: 10%;
+    border: 2px solid rgb(0, 0, 0);
+
+    .MuiDataGrid-row:hover,
+    .MuiDataGrid-cell:hover {
+        background-color: rgba(72, 245, 193, 0.2);
+        border-color: rgb(72, 245, 193);
+    }
+
+    .MuiDataGrid-row:active,
+    .MuiDataGrid-cell:active {
+        background-color: rgba(72, 245, 193, 0.2);
+        border-color: rgb(72, 245, 193);
+    }
+
+    .MuiDataGrid-root {
+        border: none;
+    }
+
+    .MuiDataGrid-columnHeaders {
+        background-color: black;
+
+        .MuiDataGrid-columnHeaderTitle {
+            color: rgb(72, 245, 193);
+            font-weight: bold;
+            text-align: center;
+            font-size: 140%;
+            font-family: "Nanum Gothic", sans-serif;
+            font-optical-sizing: auto;
+            font-style: normal;
+        }
+    }
+
+    .MuiDataGrid-cell {
+        color: #000000;
+        background-color: rgb(255, 255, 255);
+        text-align: center;
+        font-size: 110%;
+
+        &:hover {
+            border-color: rgb(72, 245, 193);
+        }
+    }
+
+    .MuiDataGrid-cellCheckbox,
+    .Mui-checked {
+        color: rgb(72, 245, 193) !important;
```

Comment:
dont use important

Reply:
I tried many ways to manage without important and nothing worked

Reply:
Did you try to check how the default color is declared in mui and use the same declaration?

Reply:
now i check it and its also not work!
if its so important i can delete it completly because  its a checkbox and its not nust,,,